src/device-api/v2.ts

```diff
@@ -327,8 +328,11 @@ export function createV2Api(router: Router) {
 					delete apps.local[app];
 				}
 			}
+
+			const commit = await commitStore.getCommitForApp(appId);
+
 			// Return filtered applications
-			return res.status(200).json(apps);
+			return res.status(200).json({ commit, ...apps });
```

Comment:
The commit from `apps` is going to override `commit` from the store? Just checking...

Reply:
There's now no longer any commit returned from the `getStatus` function